doc/source/user/select-openstack-services.rst

```diff
@@ -1,10 +1,27 @@
-================================
-Select OpenStack services to run
-================================
+=========================
+Select OpenStack Services
+=========================
 
 SUSE Containerized OpenStack currently deploys OpenStack Cinder,
 Glance, Heat, Horizon, Keystone, Neutron, Nova.
 
-You can decide which services to deploy by:
+SUSE Containerized OpenStack deployment will automatically add the following
+host rules to the /etc/hosts file on the deployer:
 
-TODO.
+.. code-block:: console
+
+   172.16.1.100 identity.openstack.svc.cluster.local
```

Comment:
I guess the IPs are just examples? Maybe mention that? Or is that really hardcoded?

Reply:
I agree, we should mention that this is an example IP address

Reply:
IPs changed to correspond with other IP address examples